.husky/pre-commit

```diff
@@ -1,4 +1,20 @@
-#!/usr/bin/env sh
+#!/bin/sh
+
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm run lint
+TARGET_FOLDER="packages/distributor/solana/generated" 
```

Comment:
tbh I don't think it should be a pre commit hook specifically. It is a one time operation after a folder update. Pre commit hooks must not change code logic; how is dev supposed to check validity if commit+push is too common an operation?

Another thing: how to scale this const to multiple sources? 

more security: plans on banning BN.js imports? https://eslint.org/docs/latest/rules/no-restricted-imports

Reply:
The pre-commit hook is there so dev doesn't manually have to convert BN to BigNumber once generating code - I guess I could just make it a script that can be manually run.

I think a wildcard like "*/generated/*" or iterating through all folders named "generated" would work

We can go with banning BN.js imports